SubtitlesParser/Classes/Parsers/SrtParser.cs

```diff
@@ -81,6 +81,8 @@ public List<SubtitleItem> ParseStream(Stream srtStream, Encoding encoding)
                         {
                             // we found the timecode, now we get the text
                             item.Lines.Add(line);
+                            // strip formatting by removing anything within curly braces or angle brackets, which is how SRT styles text according to wikipedia (https://en.wikipedia.org/wiki/SubRip#Formatting)
+                            item.PlaintextLines.Add(Regex.Replace(line, @"\{.*?\}|<.*?>", string.Empty));

```

Comment:
Why maintaining two lists (item.Lines and item.PlaintextLines) instead of computing item.PlaintextLines when accessing the getter (the result can be cached if performance is an issue but I don't think it should be)

Reply:
I was originally going to do it this way, but as I was planning it out I realized that would require a reference to what file format the SubtitleItem is for, and/or format specific parser code in the SubtitleItem class. Neither of which really felt "right", so I opted to keep the parser logic in the parser, and just have it populate when it's parsing. 

It could be done that way (in a way that feels "right" -- at least to me) where SubtitleItem is a abstract class, and have other classes like SsaSubtitleItem and SrtSubtitleItem, but I felt that was overkill.

Reply:
That's a good point. We can leave this as is for now. 
I'm wondering whether we would really need to keep track of the file format. In theory, subtitle item can have formatting elements hence the use of this new property you introduced (PlaintextLines). Some subtitles files do not contain formatting elements, only plain text, but the overhead shouldn't be substantial.
When I have a little bit of time, I'll take a deeper look at it.